src/main/java/com/wrapper/ResponseController/OnConfirmResponse.java

```diff
@@ -0,0 +1,96 @@
+//
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by FernFlower decompiler)
+//
+
+package com.wrapper.ResponseController;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.io.Serializable;
+
+public class OnConfirmResponse implements Serializable {
+	private String requestId;
+	private String timestamp;
+	private AuthData auth;
+	private String error;
+	private RespData resp;
+
+	public OnConfirmResponse() {
+	}
+
+	public String getRequestId() {
+		return this.requestId;
+	}
+
+	public void setRequestId(String requestId) {
+		this.requestId = requestId;
+	}
+
+	public String getTimestamp() {
+		return this.timestamp;
+	}
+
+	public void setTimestamp(String timestamp) {
+		this.timestamp = timestamp;
+	}
+
+	public AuthData getAuth() {
+		return this.auth;
+	}
+
+	public void setAuth(AuthData auth) {
+		this.auth = auth;
+	}
+
+	public String getError() {
+		return this.error;
+	}
+
+	public void setError(String error) {
+		this.error = error;
+	}
+
+	public RespData getResp() {
+		return this.resp;
+	}
+
+	public void setResp(RespData resp) {
+		this.resp = resp;
+	}
+
+	public String printData() {
```

Comment:
lets just log and not print any thing..we can set log level as info and debug but that can be done later as well and not on priority